src/app/static/src/tests/components/header/userHeader.test.ts

```diff
@@ -34,4 +37,23 @@ describe("user header", () => {
         const bugLink = wrapper.find("a[href='https://forms.gle/QxCT1b4ScLqKPg6a7']");
         expect(bugLink.text()).toBe("Report a bug");
     });
+
+    it("computes language", () => {
+        const wrapper = shallowMount(UserHeader, {localVue, store});
+        const vm = (wrapper as any).vm;
+        const result = vm.currentLanguage;
+        expect(result).toStrictEqual("EN");
+    });
+
+    it("contains help document link", () => {
+        const store = new Vuex.Store({
+            state: {language: Language.en}
```

Comment:
Language defaults to English anyway so shouldn't really need this, but I guess you're using a different store to the one used by all the other tests because you're changing its value. 

I might be inclined to use the file-level store to test the default (English) link, and make a local frenchStore to test another wrapper with the French version. 

Or at least to rename the local variable 'store' here to languageStore or something, so it's not confusable with the file level one (would then need to shallow mount as eg {store: languageStore}).

Reply:
I've updated this now, it requires setting the translations though after switching to French which isn't super nice. 

Is there a junit `@Before` equivalent for this test framework? I could use that to make sure `registerTranslations` is called with English `store` before every test?

Reply:
Right, yes, sorry, I had forgotten that registerTranslations affects the translation module state, so of course you need to re-register. You can use beforeEach: https://jestjs.io/docs/en/setup-teardown